lib/parallel_tests.rb

```diff
@@ -25,6 +20,18 @@ def determine_number_of_processes(count)
       ].detect{|c| not c.to_s.strip.empty? }.to_i
     end
 
+    def pids
+      @pids ||= Pids.new(pid_file_path)
+    end
+
+    def pid_file_path
```

Comment:
private ?

Reply:
`pid_file_path` is used by the runner to set the env: `"PARALLEL_PID_FILE" => ParallelTests.pid_file_path` so I don't think it can be private. However `new_pid_file_path` can.